pages/home.md

```diff
@@ -48,13 +48,13 @@ permalink: /
         <div class="columns">
           <div class="column is-hidden-touch is-offset-1 is-4">
             <figure class="image">
-              <img src="{{site.baseurl}}/assets/img/like-peas-in-a-pod.svg" alt="[]" />
+              <img src="{{site.baseurl}}/assets/img/single-sign-on.svg" alt="[]" />
             </figure>
           </div>
           <div class="column is-offset-1 is-5">
-            <h2 class="title">Like peas in a Pod</h2>
+            <h2 class="title">The last login you'll ever need</h2>
             <p class="content is-large">
-              Your <a href="{{site.baseUrl}}/faqs#pod" title="Frequently Asked Questions - what is a Pod?">Pod</a> is where you store your data. Pods and apps are separate but compatible so you are free to change apps while keeping your data. For example, if you want to switch messaging app you can take your contacts and chat history along easily, as long as both apps support Solid.
+              Solid provides for the first time a <b>single global logon system</b>, so that when you log into any web site, instead of having  to log in with the usual 'f' and 'g' blue buttons, and then be tracked by Facebook and Google, you can log in with any Solid provider you trust, and that won't track you.
```

Comment:
Not sure if we want to directly confront f and g.

Reply:
Modified, to make them just examples